frontend/pages/admin/pacta-version/[id].vue

```diff
@@ -8,9 +8,27 @@ const { fromParams } = useURLParams()
 
 const id = presentOrCheckURL(fromParams('id'))
 
-const prefix = 'admin/pacta-version/[id]'
-const persistedPactaVersion = useState<PactaVersion>(`${prefix}.persistedPactaVersion`)
+const prefix = `admin/pacta-version/${id}`
 const pactaVersion = useState<PactaVersion>(`${prefix}.pactaVersion`)
+const { data: persistedPactaVersion, error, refresh } = await useAsyncData(`${prefix}.getPactaVersion`, () => {
+  return withLoadingAndErrorHandling(() => {
+    return pactaClient.findPactaVersionById(id)
+      .then(handleOAPIError)
+  }, `${prefix}.getPactaVersion`)
+})
+if (error.value) {
+  throw createError(error.value)
+}
+if (!persistedPactaVersion.value) {
```

Comment:
We can make this a `present` check, maybe `presentOrCheckURL`? 

Same below on line 28

Reply:
GC, done. I do think we lose relevant context doing so, so I'm going to add an optional third parameter that specifies context.

Update: I went a bit further and merged `present` and the remediation error stuff with Nuxt's `createError`, take another look and let me know what you think. I think standardizing on the Nuxt stuff will make central error reporting (and modal customizing) much easier, and we get to keep strong typing on the error metadata here too